ui/guest/package.json

```diff
@@ -81,7 +81,7 @@
     ]
   },
   "scripts": {
-    "build": "rollup -c",
+    "build": "rollup -c --filterLogs \"code:A_COMMENT\" ",
```

Comment:
Added filterLogs due to a warning because of some comments (check logs in [here](https://gist.github.com/jvega190/88a47cbd432a7d4e09cb96c1bed4be07)). Please check at the end of the General Changes section in [here ](https://rollupjs.org/migration/#general-changes)

Reply:
Leave the warnings.

You need to be more specific on what "...the end of the general changes...". Link to the specific sections and mention what's the point of reviewing.